app/controllers/items_controller.rb

```diff
@@ -0,0 +1,47 @@
+class ItemsController < ApplicationController
+  def index
+    @merchant = Merchant.find(params[:merchant_id])
+    @items = @merchant.items
```

Comment:
This works!  It would be perfectly ok to call `@merchant.items` in your view as well (instead of storing `@items` in an instance variable) and it would DRY up the controller a little bit.

Reply:
Good call!